app/controllers/items_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ItemsController < ApplicationController
+  before_action :start_facade, only: [:show]
```

Comment:
I like where you were going with this. I think instead I would call `start_facade` directly in your show action, especially since you are only using this `before_action` for show.